nsxt/resource_nsxt_policy_firewall_exclude_list_member.go

```
@@ -0,0 +1,144 @@
+	if 0 <= memberInList(id, obj.Members) {
+		return true, nil
+	}
+
+	return false, nil
+}
+
+func resourceNsxtPolicyFirewallExcludeListMemberCreate(d *schema.ResourceData, m interface{}) error {
+	connector := getPolicyConnector(m)
+	member := d.Get("member").(string)
+
+	doUpdate := func() error {
+		var obj model.PolicyExcludeList
+
+		client := security.NewExcludeListClient(getSessionContext(d, m), connector)
+		obj, err := client.Get()
+		if isNotFoundError(err) {
+			obj = model.PolicyExcludeList{
+				Members: []string{member},
+			}
+		} else if err != nil {
+			return err
+		}
+		if 0 <= memberInList(member, obj.Members) {
+			return errors.AlreadyExists{}
+		}
+		obj.Members = append(obj.Members, member)
+		_, err = client.Update(obj)
+		if err != nil {
+			return err
+		}
+
+		d.SetId(member)
+
+		return nil
+	}
+	commonProviderConfig := getCommonProviderConfig(m)
+	err := retryUponPreconditionFailed(doUpdate, commonProviderConfig.MaxRetries)
+	if err != nil {
+		return handleCreateError("PolicyFirewallExcludeListMember", member, err)
+	}
+
+	return resourceNsxtPolicyFirewallExcludeListMemberRead(d, m)
+}
+
+func resourceNsxtPolicyFirewallExcludeListMemberRead(d *schema.ResourceData, m interface{}) error {
+	connector := getPolicyConnector(m)
+	member := d.Id()
+
+	client := security.NewExcludeListClient(getSessionContext(d, m), connector)
+	obj, err := client.Get()
+	if err != nil {
+		return handleReadError(d, "PolicyFirewallExcludeListMember", member, err)
+	}
+	if 0 > memberInList(member, obj.Members) {
+		return errors.NotFound{}
+	}
+	d.Set("member", member)
+	return nil
+}
+
+func resourceNsxtPolicyFirewallExcludeListMemberDelete(d *schema.ResourceData, m interface{}) error {
+	connector := getPolicyConnector(m)
+	member := d.Get("member").(string)
+
+	doUpdate := func() error {
+		var obj model.PolicyExcludeList
+
+		client := security.NewExcludeListClient(getSessionContext(d, m), connector)
+		obj, err := client.Get()
+		if isNotFoundError(err) {
```

We should silently swallow the error in `not found` case